File: src/test/resources/integration_tests/python/mytest.py
# Import the VertxAssert class - this has the exact same API as JUnit
from io.vertx.testtools import VertxAssert

import vertx

def test_1() :
    VertxAssert.testComplete()

def test_2() :
    VertxAssert.testComplete()

# At the end of your test script make sure call init_tests
import vertx_tests
vertx_tests.init_tests(locals())

File: src/test/resources/integration_tests/ruby/mytest.rb
# Always require "vertx_tests" at the top of your test script
require "vertx_tests"

require "vertx"

# The test methods must begin with "test"

def test1
  VertxAssert.testComplete()
end

# And make sure you call init_tests at the bottom
init_tests(self)


